src/pages/Login.js

```diff
@@ -3,12 +3,14 @@ import React from 'react';
 import logo from '../trivia.png';
 import '../App.css';
 import Form from '../components/Form';
+import Setting from '../components/Setting';
```

Comment:
Normalmente eu costumo pular uma linha entre as importações de imagens e biblioteca para os componentes, vc acha que é uma boa?
```suggestion
import logo from '../trivia.png';
import '../App.css';

import Form from '../components/Form';
import Setting from '../components/Setting';
```